### PR TITLE
add easyconfigs for recent Mesa with intel/2014b

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@JensTimmerman: I have yet to find a detailed procedure on how that would work... I saw that message too, but couldn't find an easy way to only build libudev.

**Comment by hajgato** [comment]
What about eudev? Maybe it is even better than system-udev. see: http://www.gentoo.org/proj/en/eudev/

**Comment by boegel** [comment]
@hajgato: eudev certainly looks interesting! What kind of kernel headers does it require, exactly?

**Comment by hajgato** [comment]
@boegel I did not have time to look into the details, but I quickly ran configure, and it died with: `configure: error: *** KERNEL header not found`
Although I have kernel, kernel-dev, and kernel-headers packages installed. Next week I can look into this in more details, but I am busy this week with something else.

**Comment by boegel** [comment]
OK, thanks for the info, certainly looks like an interesting avenue.

**Comment by hajgato** [comment]
@boegel Do we have any update in this field? Finally I was able to compile eudev-3.0, although I am not entirely sure that it works. (especially with pre 2.6.34 kernel)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 21 out of 22 (22 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/888d5a85b00ccf314c4c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 22 out of 22 (22 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b01a5707b2cef070bc36 for a full test report.

**Comment by boegel** [comment]
@hajgato, @wpoely86: Mesa build now works when `eudev` from #1578 is included as a dependency \o/

**Comment by hajgato** [comment]
@boegel Now we have to check whether it actually works. And we have to polish eudev a bit.

File: easybuild/easyconfigs/m/Mesa/Mesa-10.3.4-intel-2014b-Python-2.7.8.eb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+dependencies = [
+    ('Python', pythonver),
+    ('libxml2', '2.9.2', versionsuffix),
+    ('libdrm', '2.4.58'),
+    ('libX11', '1.6.2', versionsuffix),
+    ('libXext', '1.3.3', versionsuffix),
+    ('libXfixes', '5.0.1'),
+    ('libXdamage', '1.1.4', versionsuffix),
+    ('libXfont', '1.5.0', versionsuffix),
+    ('eudev', '3.0'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/Mesa/Mesa-10.3.4-intel-2014b-Python-2.7.8.eb]: I suggest we do the same thing as we do with openssl: put an osdependency on udev and comment out the eudev easyconfig for those who really want it.

**Comment by hajgato** [comment]
Anyway, it looks that we will not be able to get rid of osdependency completely. I guess that we cannot get rid of eudev kernel-devel osdependency. (But not yet tested)

File: easybuild/easyconfigs/x/xorg-macros/xorg-macros-1.17-intel-2014b.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+homepage = 'http://cgit.freedesktop.org/xorg/util/macros'
+description = """X.org macros utilities."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = ['http://cgit.freedesktop.org/xorg/util/macros/snapshot']  # no slash ('/') at the end!
+sources = ['util-macros-%(version)s.tar.gz']
+
+builddependencies = [('Autoconf', '2.69')]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/x/xorg-macros/xorg-macros-1.17-intel-2014b.eb]: which is now called the `autotools` bundle; please adjust accordingly...

**Comment by boegel** [comment]
@hajgato: what's the latest status on this? Can we use `eudev` as a dep for Mesa?

**Comment by hajgato** [comment]
I think yes.

**Comment by boegel** [comment]
obsolete, we already have Mesa-10.5.5-intel-2015a-Python-2.7.10.eb and even Mesa-11.0.2-intel-2015b-Python-2.7.10.eb merged in

**Pull request closed without merging**